### PR TITLE
fix: package.json

### PR DESCRIPTION
**Pull request by davidfragalaureano**
Addresses [this](https://github.com/ExodusMovement/aptos-sdk-fork/pull/5#issuecomment-2364254490) issue.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,15 +7,7 @@
...
-  "exports": {
-    ".": {
-      "import": "./dist/index.mjs",
-      "require": "./dist/index.js",
-      "types": "./dist/index.d.ts"
-    }
-  },
   "files": [
     "dist/index.d.ts",
     "dist/index.mjs",
```
Comment (ChALkeR) [inline package.json]: then we don't need to include separate mjs file anymore)

and later we might switch that to just including mjs and not .js, but that would require an audit of the diff

Comment (ChALkeR) [inline package.json]:
```suggestion
```

Comment (davidfragalaureano) [reply package.json]: [`d538812`](https://github.com/ExodusMovement/aptos-sdk-fork/pull/6/commits/d538812b5dc1c61412f0058b529369d3aa70d1ec)

**Review by lanre-mark — APPROVED** [review]
utACK

**Comment by davidfragalaureano** [comment]
@ChALkeR Good to merge?

**Review by ChALkeR — APPROVED** [review]

**Review by feri42 — APPROVED** [review]

**Comment by feri42** [comment]
Who can merge this?

**Pull request merged**